src/deadline/client/cli/_groups/manifest_group.py

```diff
@@ -0,0 +1,206 @@
+    required=True,
+    help="The path to manifest file to diff against.",
+)
+@click.option(
+    "-i",
+    "--include",
+    default=None,
+    help="Glob syntax of files and directories to include in the manifest. Can be provided multiple times.",
+    multiple=True,
+)
+@click.option(
+    "-e",
+    "--exclude",
+    default=None,
+    help="Glob syntax of files and directories to exclude in the manifest. Can be provided multiple times.",
+    multiple=True,
+)
+@click.option(
+    "-ie",
+    "--include-exclude-config",
+    default=None,
+    help="Include and exclude config of files and directories to include and exclude. Can be a json file or json string.",
+)
+@click.option("--json", default=None, is_flag=True, help="Output is printed as JSON for scripting.")
+@_handle_error
+def manifest_diff(
+    root: str,
+    manifest: str,
+    include: List[str],
+    exclude: List[str],
+    include_exclude_config: str,
+    json: bool,
+    **args,
+):
+    """
+    Check file differences between a directory and specified manifest.
+    """
+    raise NotImplementedError("This CLI is being implemented.")
+
+
+@cli_manifest.command(
+    name="download",
+    help="BETA - Download Job Attachment Manifests for a Job, or Step including dependencies.",
+)
+@click.argument("download_dir")
+@click.option("--profile", help="The AWS profile to use.")
+@click.option("--job-id", required=True, help="The AWS Deadline Cloud Job to get. ")
+@click.option("--step-id", help="The AWS Deadline Cloud Step to get. ")
+@click.option("--farm-id", help="The AWS Deadline Cloud Farm to use. ")
+@click.option("--queue-id", help="The AWS Deadline Cloud Queue to use. ")
+@click.option(
+    "--json", default=None, is_flag=True, help="Output is printed as JSON for scripting. "
+)
+@_handle_error
+def manifest_download(
+    download_dir: str,
+    job_id: str,
+    step_id: str,
+    json: bool,
+    **args,
+):
+    """
+    Downloads input manifest of previously submitted job.
+    """
+    raise NotImplementedError("This CLI is being implemented.")
+
+
+@cli_manifest.command(
+    name="upload",
+    help="BETA - Uploads a job attachment manifest file to a Content Addressable Storage's Manifest store. If calling via --s3-cas-path, it is recommended to use with --profile for a specific AWS profile with CAS S3 bucket access.",
+)
+@click.argument("manifest_file")
+@click.option("--profile", help="The AWS profile to use.")
+@click.option("--s3-cas-path", help="The path to the Content Addressable Storage root.")
+@click.option(
+    "--farm-id", help="The AWS Deadline Cloud Farm to use. Alternative to using --s3-cas-path."
+)
+@click.option(
+    "--queue-id", help="The AWS Deadline Cloud Queue to use. Alternative to using --s3-cas-path."
+)
+@click.option("--json", default=None, is_flag=True, help="Output is printed as JSON for scripting.")
+@_handle_error
+def manifest_upload(
+    manifest_file: str,
+    s3_cas_path: str,
+    json: bool,
+    **args,
+):
```

Comment:
super nit: comment on what it does much like the other commands

Reply:
I will do this when the upload CLI is added in the next set of PRs.

Reply:
There is a comment above though - 
`@cli_manifest.command(
    name="upload",
    help="BETA - Uploads a job attachment manifest file to a Content Addressable Storage's Manifest store. If calling via --s3-cas-path, it is recommended to use with --profile for a specific AWS profile with CAS S3 bucket access.",
)`